CTP/shell/src/com/navercorp/cubridqa/shell/result/FeedbackFile.java

```diff
@@ -93,6 +100,73 @@ public void onTaskStopEvent() {
 	public void setTotalTestCase(int tbdNum, int macroSkippedNum, int tempSkippedNum) {
 		println("The Number of Test Cases: " + tbdNum + " (macro skipped: " + macroSkippedNum + ", bug skipped: " + tempSkippedNum + ")");
 	}
+	
+	public void showTestResult(){
```

Comment:
It's better not to analyze the logs in order to show test result.

Create a structurized file (e.g.  .test_status_data) to save status.